dll/win32/kernel32/client/compname.c

```diff
@@ -31,6 +32,13 @@
 #define NDEBUG
 #include <debug.h>
 
+typedef enum _DNS_NAME_FORMAT
```

Comment:
Why defining it here instead of using including an appropriate header?

Reply:
Used `<windns.h>`. Done.